Vision/tests/Unit/V1/ImageAnnotatorClientExtensionTest.php

```diff
@@ -115,7 +116,12 @@ public function testAnnotateImageWithImageContext()
         $this->transport->startUnaryCall( Argument::allOf(
                     Argument::type(Call::class),
                     Argument::which('getMethod', 'google.cloud.vision.v1.ImageAnnotator/BatchAnnotateImages'),
-                    Argument::which('getMessage', $expectedMessage)
+                    Argument::that(function($arg) use ($expectedMessage) {
```

Comment:
What do you think about adding a prophecy token to do this equivalence check? Similar to [ArrayHasSameValuesToken](https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/Core/src/Testing/ArrayHasSameValuesToken.php), but with the guts of [GeneratedTest](https://github.com/tmatsuo/gax-php/blob/affac4628cb1bb83582e5ea7231e64fa5cf49d4c/src/ApiCore/Testing/GeneratedTest.php)?

Reply:
@jdpedrie Potentially it's a nice addition, but I'm ok with the current changes in our test code.

Does having the `TokenInterface` simplify the hack here?

Reply:
I think it could just simplify this block a bit (and others, if other tests perform similar checks.

````php
Argument::allOf(
    new ProtobufEqualsToken($arg, $expectedMessage)
)
```

vs. the current `Argument::that()` and callback. I'm ok to proceed without it, just something I wanted to bring up.